src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml

```diff
@@ -40,6 +39,25 @@
                  </DataTrigger>
             </Style.Triggers>
         </Style>
+        <Style x:Key="ConnectionTypeComboBoxStyle"
+               TargetType="{x:Type ComboBox}">
+            <Style.Triggers>
+                <DataTrigger Binding="{Binding SupportSSHConnections}"
+                             Value="true">
+                    <Setter Property="Grid.ColumnSpan"
+                            Value="1" />
+                    <Setter Property="IsEnabled"
+                            Value="True" />
+                </DataTrigger>
```

Comment:
nit: Do you need this? I think you could at least get rid of this data trigger and just have these be the default setters for the style